base.py

```diff
@@ -178,45 +192,144 @@ def run(self):
+                return False
+
+        return True
+
+    def connect_to_browser(self):
+        if os.path.isfile('/.docker'):
+            # xvfb wrapper starting
+            print('Using Xvfb')
+            self.xvfb = Xvfb()
+
+            try:
+                self.xvfb.start()
+
+            except Exception:
+                return False
+
+        options = Options()
+        options.add_argument('--no-sandbox')
+
+        try:
+            self.browser = Browser('chrome', headless=False, options=options,
+                                   wait_time=30,
+                                   executable_path='./drivers/chromedriver')
+
+        except Exception:
+            print('\nFailed to connect with browser.')
+            return False
+
+        self.browser.driver.implicitly_wait(self.sticky_timeout)
+        self.browser.driver.set_page_load_timeout(self.page_load_timeout)
+        self.browser.driver.set_window_size(*self.browser_window_size)
+        self.browser.visit(self.addr)
+
+        return True
+
+    def close_browser_connection(self):
+        if os.path.isfile('/.docker'):
+            self.xvfb.stop()
+
+        return True
+
+    def setup(self):
+        pass
+
+    def tear_down(self):
+        pass
 
 
 class RocketChatTestCase(SplinterTestCase):
-    def __init__(self, addr, username, password, create_test_user=True,
-                 **kwargs):
+    def __init__(self, addr, username, password, expected_rooms=None, **kwargs):
         SplinterTestCase.__init__(self, addr, **kwargs)
 
-        self.rocket = RocketChat(username, password, server_url=addr)
+        if expected_rooms:
+            self.expected_rooms = expected_rooms.split(',')
 
         self.schedule_pre_test_case('login')
         self.schedule_pre_test_case('test_check_version')
 
-        if create_test_user:
-            self.schedule_pre_test_case('create_user')
-
         self.username = username
         self.password = password
         self._rc_version = '0.70'
 
+        self.test_user_id = None
         self.test_username = 'noname'
         self.test_full_name = 'No Name'
         self.test_email = 'noname@nodomain.com'
         self.test_password = 'pass'
 
-        if create_test_user:
-            self.schedule_test_case('remove_user')
+    def connect_to_rc_api(self):
+        try:
+            self.rocket = RocketChat(
+                self.username,
+                self.password,
+                server_url=self.addr
+            )
+
+        except RocketAuthenticationException:
+            print('Rocket.Chat auth error. Incorrect username or password.')
+            return False
+
+        except RocketConnectionException:
+            print('Could not connect to Rocket.Chat API.')
+            return False
+
+        return True
+
+    def delete_all_extra_users(self):
+        all = self.rocket.users_list().json()
+        to_be_deleted = [i['_id'] for i in all['users'] if 'user' in i['roles']]
```

Comment:
And line length is too long. Maybe you will cut it?
[PEP8 about line lenght](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)